RiotSwiftUI/Modules/Room/PollHistory/View/PollListItem.swift

```diff
@@ -0,0 +1,76 @@
+// 
+// Copyright 2023 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+struct PollListData {
+    let startDate: Date
+    let question: String
+}
```

Comment:
Should probably go in `PollHistoryModels` fwiw

Reply:
I fell like dumb data model can stay in the same file of the view that consumes them.
Doing so you have everything you need to work on the component in a single file.
I'm not against moving it btw.